benchmarks/src/main/java/com/slack/kaldb/IndexingBenchmark.java

```diff
@@ -83,6 +88,9 @@ public void createIndexer() throws Exception {
     DocumentBuilder<LogMessage> documentBuilder = LogDocumentBuilderImpl.build(false);
 
     luceneDocument = documentBuilder.fromMessage(logMessage);
+
+    String apiLogFile = System.getProperty("jmh.api.log.file", "api_logs.txt");
```

Comment:
Can we get an updated README for benchmarks with this? Maybe an example valid log line, and how to run with a specific file?